deepr/jobs/copy_dir.py

```diff
@@ -12,15 +12,22 @@
 
 @dataclass
 class CopyDir(base.Job):
-    """Copy Directory"""
+    """Copy Directory. Overwrite destination by default"""
 
     source: str
     target: str
     skip_copy: bool = False
+    overwrite_destination_folder = True
 
     def run(self):
         if self.skip_copy:
             LOGGER.info(f"NOT COPYING {self.source} to {self.target} (skip_copy=True)")
             return
+        if self.overwrite_destination_folder:
+            try:
```

Comment:
better to do an `if Path().is_dir()` rather than try / catch

Reply:
done